datahub-graphql-core/src/main/resources/timeline.graphql

```diff
@@ -0,0 +1,146 @@
+extend type Query {
+    """
+    Get list of schema blame
+    """
+    listSchemaBlame(input: ListSchemaBlameInput!): ListSchemaBlameResult
+}
+
+type TimelineConfig {
+    """
+    Whether the timeline feature is enabled.
+    """
+    enabled: Boolean!
+}
+
+"""
+Enum of SemanticVersionChange
+"""
+enum SemanticVersionChangeType {
+    NONE
+    PATCH
+    MAJOR
+    MINOR
+    EXCEPTIONAL
+}
+
+
+"""
+Enum of types of changes
+"""
+enum ChangeOperationType {
+    ADD
+    MODIFY
+    REMOVE
+}
+
+"""
+Enum of CategoryTypes
+"""
+enum ChangeCategoryType {
+    DOCUMENTATION
+    GLOSSARY_TERM
+    OWNERSHIP
+    TECHNICAL_SCHEMA
+    TAG
+}
+
+"""
+Input for getting list of latest schema changes computed at a specific version.
+"""
+input ListSchemaBlameInput {
+    """
+    The dataset urn
+    """
+    datasetUrn: String!
+    """
+    Categories to filter on
+    """
+    categories: [ChangeCategoryType!]
```

Comment:
If this is not provided, what are the defaults? I notice its optional

Reply:
Are categories actually used anywhere?

Reply:
Just TECHNICAL_SCHEMA, but we can change this in the future.

Reply:
pls document the default behavior since this is a public api

Reply:
Done!